.github/workflows/release_windows_pytorch_wheels.yml

```diff
@@ -16,17 +51,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        config:
-          - { AMDGPU_FAMILIES: gfx110X-dgpu }
-          # TODO(scotttodd): enable once nightly builds include these
-          # - { AMDGPU_FAMILIES: gfx1151 }
-          # - { AMDGPU_FAMILIES: gfx120X-all }
-        python_version: ["3.11", "3.12"]
+        python_version: ["3.11", "3.12", "3.13"]
 
     uses: ./.github/workflows/build_windows_pytorch_wheels.yml
```

Comment:
I recently revised this workflow further (PR #998) and expect for this particular line the workflows are identically. Could merge them and rather add an input parameter that tells if to build for Linux or Windows maybe?.

Reply:
Well we can go for it as is, this isn't blocking but wanted to flag that we can clean in a follow up :)